spritz/README.md

```diff
@@ -0,0 +1,153 @@
+# spritz
+
+_Animate your view pager with Lottie and zero efforts._
+
+--------
+
+Spritz is an Android library to seamlessly trigger a Lottie animation when the user drags a `ViewPager` or changes page. It supports animating 
+while swiping and auto-playing when the swipe is complete.
+
+![A taste of spritz](images/a-taste-of-spritz.gif)
+
+## How to use
+
+### Basics
+
+Each page in the `ViewPager` has two possible animations:
+
+- a "swipe away" animation that will be triggered while the user drags from one page to the next or previous one
+- an "autoplay" animation segment that plays as soon as the user enters the destination page
+
+When swiping forward, the animation will stop right before auto-playing. When swiping backwards, instead, the animation will stop at the end of the
+previous page's autoplay animation, so it doesn't play again since that page is in an "already visited" state.
+
+### Lottie Animation
+
+To learn how to create a Lottie-compatible animation from After Effects, please read 
+[our blog post "Whole Lottie Love"](https://www.novoda.com/blog/whole-lottie-love/), which contains basic instructions on how to export an animation
+into a Lottie-supported format.
+
+Your animation should have:
+
+- for each page, a "swipe away" animation
+- for each page, optionally, an "autoplay" animation segment
+
+These segments must be consecutive to one another, as if the user continuously moves to one page to another.
+
+### Code!
+
+After getting your `LottieAnimationView` and your `ViewPager`, just create a `Spritz` object with the following syntax:
+
+```
```

Comment:
Can you put the language here so it does syntax highlighting?

```
  ```java
```

Same elsewhere.

Reply:
Done in 7adc4638ab2c0b1e585c94fd305ce220724b2e61.